client/src/components/inventory/RequestButton.tsx

```diff
@@ -43,9 +43,9 @@ function RequestButton({requestedItem, user, toastManager}: RequestButtonProps)
         >
           {
               ({loading, error, data}: any) => {
-                // if (loading) {
-                //     return <Loader active inline="centered" content="Just a sec!"/>;
-                // }
+                if (loading) {
+                    return <Loader active inline="centered" content="Just a sec!"/>;
```

Comment:
Make sure you test this, it was commented out because iirc it didn't look right.  but feel free to fix it if you can

Reply:
It works fine for me. I can update it stylistically in a different PR, just wanted to get this one and the next couple out first